docker/deploy-fedimintd/.env

```diff
@@ -1,18 +1,18 @@
 # Bitcoin auth information
 # generate with https://jlopp.github.io/bitcoin-core-rpc-auth-generator/,
 # default is user: bitcoin, pass: bitcoin
-BITCOIND_RPC_AUTH=bitcoin:54ae356e13a76dc8068e960eb43193cb\$efeeb347a1f0b4a7b7832cc26e68861bc46f89126a8e136ff9932cad47739041
+BITCOIND_RPC_AUTH=bitcoin:54ae356e13a76dc8068e960eb43193cb$efeeb347a1f0b4a7b7832cc26e68861bc46f89126a8e136ff9932cad47739041
 
 # This domain should point to the machine fedimintd is being deployed to
 FM_DOMAIN=my-super-host.com
 
 # Where bitcoind is reachable
 FM_BITCOIN_RPC_KIND=bitcoind
-FM_BITCOIN_RPC_URL=http://bitcoin@bitcoin@127.0.0.1:8332
+FM_BITCOIN_RPC_URL=http://bitcoin:bitcoin@bitcoin:8332
```

Comment:
This is called from the `fedimintd` container, so we can't access bitcoind at localhost. Instead, we can reach it with the container's name `bitcoin`.